README.md

```diff
@@ -1,2 +1,31 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+
+##basic word similarity guessing pseudocode
+
+##the idea is that one person enters the secret word and then gives the device to someone else to guess
+
+secretWord <- get input
+wordLength <- length of 'secretWord'
+attempts <- 10
+success <- FALSE
+
+print("guess the", wordLength, "letter word")
+
+while attempts is greater than 0 AND success is FALSE:
+>	likeness <- 0
+>	guess <- get input
+>	for i in range wordLength:
```

Comment:
`range` might not be understood by a non-Python programmer. Would be better to try and make it more explicit, e.g.
```
for i = 1 to wordLength
```
or
```
for i = 1, 2, ..., wordLength
```